demo/index.html

```diff
@@ -0,0 +1,120 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+        <title>kwc-feature-list demo</title>
+
+        <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+        <link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">
+        <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
+        <link rel="import" href="../kwc-feature-list.html">
+
+        <custom-style>
+            <style is="custom-style" include="demo-pages-shared-styles">
+                :root {
+                    --kwc-feature-list-bg-color: #e9ebec;
+                }
+            </style>
+        </custom-style>
+    </head>
+    <body>
+        <div class="vertical-section-container">
+            <h3>Four features with a title</h3>
+            <demo-snippet>
+                <template id="four" is="dom-bind">
+                    <kwc-feature-list features="[[features]]"
+                                      title="[[title]]"></kwc-feature-list>
```

Comment:
We never decided if we want to use the closing tag at the same line or on a new line. I would advocate to do it on a new line just so it's easier to fit within the 80 columns for deeper html structures. In the past I have already missed and deleted a property line that has a closing tag and it was pretty sad.